P9Operators/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿
+// Execise 10
+/*
+int TwentyFourHourSeconds = 86400;
+
+Console.WriteLine("Give me a number of sec.");
+float userInput = float.Parse(Console.ReadLine());
+
+// in total days form
+float inTotal = userInput / TwentyFourHourSeconds;
+
+// days, hour, min & sec
+int intInput = (int)userInput;
+
+int day = intInput / TwentyFourHourSeconds;
+int oneDayHours = day * 24;
+int hours = intInput / 60 / 60 - 24;
+int minutes = intInput / 60 - ( hours + oneDayHours) * 60;
+int seconds = intInput - (((hours + oneDayHours) * 60 * 60) + (minutes * 60));
```

Comment:
Overall, you have caused yourself a little too much headache by not using the Modulo Operator. There was another exercise where you had to use the Modulo operator. (Exercise 9 on the bottom of this script)

Compare that solution:
```csharp
Console.WriteLine("Give an amount of seconds:");
int seconds = int.Parse(Console.ReadLine());

int spare = seconds % 60;
int minutes = seconds / 60;

Console.WriteLine($"{seconds} = {minutes} minute(s) and {spare} second(s)");
```

But somehow, you've done it completely differently here, and got very close to the correct solution, to be honest, but you've been slightly off due to making it a little too complicated.